src/User/Infra/Doctrine/Resources/dist-mapping/User.Entity.Credential.Email.orm.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
     <embeddable name="MsgPhp\User\Entity\Credential\Email">
-        <field name="email" unique="true" />
+        <field name="email" unique="true" length="%msgphp.doctrine.key_max_length%" />
```

Comment:
if we have `Doctrine\MappingConfig` any property should be accessible using e.g. `%msgphp.doctrine.mapping.key_max_length%`.